libraries/cyclestreets-view/src/main/res/values/strings.xml

```diff
@@ -137,5 +137,9 @@
   <string name="perm_justification_read_contacts" translatable="false"><![CDATA[
     <li>&nbsp; allow you to find the addresses of your contacts on the map, and create routes to these locations</li>
   ]]></string>
+  <string name="show_remaining_time">Show remaining time</string>
+  <string name="show_ETA">Show estimated time of arrival</string>
+  <string name="remaining">Remaining</string>
+  <string name="ETA">/ ETA</string>
```

Comment:
Perhaps a better string identifier would be "or_ETA" given that you include a slash in the text value.

Reply:
It's not really an "or", it's a separator from "Remaining".  I could use "&", but I think "/" is neater.  So I've changed the name to "slash_ETA".